framework/arcane-framework/src/test/scala/models/CdmParserTests.scala

```diff
@@ -0,0 +1,29 @@
+package com.sneaksanddata.arcane.framework
+package models
+
+import models.cdm.CSVParser
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.matchers.should.Matchers.should
+import org.scalatest.prop.TableDrivenPropertyChecks.*
+
+
+class CdmParserTests extends AnyFlatSpec with Matchers {
+
+  private val validCsvLines = Table(
```

Comment:
It seems that this method does not handle incorrect CSV files [as it did in C#](https://github.com/SneaksAndData/arcane-framework/blob/8ffdc749f044d544605a2bcd7faee8d84b5e59f6/test/Operations/CsvOperationsTests.cs#L15).

Reply:
yeah will add before merge